ansible/roles/tomcat/tasks/main.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: Creates directory for tomcat download
+  file:
+    path: "{{ tomcat_download_dir }}"
+    state: directory
+    owner: '{{work_user}}'
+  become: true
+
+- name: Creates directory for tomcat installation
+  file:
+    path: "{{ tomcat_install_dir }}"
+    state: directory
+    owner: '{{work_user}}'
+  become: true
+
+- name: Download tomcat.
+  get_url:
+    url: "{{ tomcat_download_url }}"
+    dest: "{{ tomcat_download_dir }}/apache-tomcat-{{ tomcat_version }}.tar.gz"
+
+
+- name: Expand tomcat.
+  unarchive:
+    src: "{{ tomcat_download_dir }}/apache-tomcat-{{ tomcat_version }}.tar.gz"
+    dest: "{{ tomcat_install_dir }}"
+    remote_src: true
+    creates: "{{ tomcat_install_dir }}/apache-tomcat-{{ tomcat_version }}/README.txt"
+
+
+- name: Copy tomcat file
+  template:
+    src: "../files/server.xml"
+    dest: "{{ tomcat_install_dir }}/apache-tomcat-{{ tomcat_version }}/conf/server.xml"
+    follow: yes
+
+
+- name: Adjust tomcat config
+  lineinfile:
+    path: "{{ tomcat_install_dir }}/apache-tomcat-{{ tomcat_version }}/conf/tomcat-users.xml"
+    line: '<user username={{ dspace_user }} password={{ dspace_password }} roles="admin-script,manager-gui,dspace"/>'
```

Comment:
how can we do that? like this:
'<user username= {{tomcat_user}} password={{ tomcat_password }} roles="admin-script,manager-gui,dspace"/>' ?

Reply:
@mirjan-hoffmann . Please reply to this question? I will fix immediately.

Reply:
> 

In the documentation it is mentioned to use this way:

![Screenshot from 2022-03-14 13-08-56](https://user-images.githubusercontent.com/99397044/158169381-c38d64b1-0869-4598-834c-c030f3102d1a.png)

Reply:
Yes, please use `tomcat_user` / `tomcat_password` as suggested

Reply:
OR use the following 

tomcat_user: {{ work_user }}
tomcat_password: "password"

?

Reply:
Since this is just a default-value, just use it as suggested above (the important thing is that we can adjust it for prod-installations)

> how can we do that? like this:
'<user username= {{tomcat_user}} password={{ tomcat_password }} roles="admin-script,manager-gui,dspace"/>' ?
>
> And then in vars/main.yml add this:
>
> ```
> tomcat_user: dspace
> tomcat_password: dspace
> ```